packages/creator-hub/renderer/src/modules/store/deployment/slice.ts

```diff
@@ -130,7 +131,8 @@ export const deploy = createAsyncThunk(
   async (deployment: Deployment, { dispatch, getState, rejectWithValue }) => {
     const { path, info, identity, wallet, chainId } = deployment;
     const triedServers = new Set<string>();
-    let retries = getCatalystServers(chainId).length;
+    const isWorldDeployment = info.isWorld;
+    let retries = isWorldDeployment ? 1 : getCatalystServers(chainId).length;
```

Comment:
np: wdyt about using:
```suggestion
    let retries = info.isWorld ? 1 : getCatalystServers(chainId).length;
```
?